README.md

```diff
@@ -29,6 +29,10 @@ To publish to maven local:
 ```
 ./gradlew -b buildSrc/build.gradle.kts publishToMavenLocal
 ```
+To publish:
+```
+./gradlew -b buildSrc/build.gradle.kts publish
```

Comment:
Does it make sense to have a `RELEASING.md` file with also the instructions on how to setup the environment?